### ISSUE TITLE
[Success Image Report]: Win10_22H2_EnglishInternational_x64v1.iso

### ISSUE THREAD

**Issue by kronicd** [ventoy/Ventoy#2506]
### Official Website List

- [X] I have checked the list in official website and the image file is not listed there.

### Ventoy Version

1.0.94

### BIOS Mode

Both

### Partition Style

MBR

### Image file name

Win10_22H2_EnglishInternational_x64v1.iso

### Image file checksum type

MD5

### Image file checksum value

e249522b51e9a0c6129d5e46a08b8e97

### Image file download link (if applicable)

https://software.download.prss.microsoft.com/dbazure/Win10_22H2_EnglishInternational_x64v1.iso

### Test environment

Lenovo Thinkpad 5th Gen

### More Details?

This image file booted successfully in Ventoy.

**Comment by ventoy** [comment]
Thanks!

**Comment by DAGAMES386** [comment]
The image file download link appears to be broken. Does anyone have another link?

**Comment by stlyash** [comment]
Summary: Can I use windows 11 as a live image from ventoy, instead of installing it? (If anyone can provide me with a link of any tutorial, it would be helpful)

Hello,
I wanted to ask something,
I wanted to use windows live from my ventoy flash drive, without installing it on my laptop. Is it possible?
my main os is also windows 11 

I tried opening the iso from ventoy, but I got a prompt after which i had two options, install windows and exit and I got scared and exited from that dialog. Because I didn't wanted to erase my entire OS.
